### PR TITLE
Fix search binary pagination failure

### PR DESCRIPTION
**Pull request by XuanYang-cn**
/hold
#22168

### CODE REVIEW

File: tests/python_client/testcases/test_search.py [hunk 1]
```diff
@@ -3946,10 +3946,21 @@ def test_search_binary_with_pagination(self, offset, auto_id):
...
+        unique_a, unique_b = set(search_res[0].ids), set(res[0].ids[offset:])
+
+        try:
+            assert 0 == len(unique_a-unique_b)
+            assert set(search_res[0].ids) == set(res[0].ids[offset:])
+        except Exception:
+            print(f"ids: {search_res[0].ids}")
+            print(f"distances: {search_res[0].distances}")
+            print("===========")
+            print(f"pagination ids: {res[0].ids[offset:]}")
```
Comment (zhuwenxing) [inline tests/python_client/testcases/test_search.py:3960]: using log.info() is better

**Review by zhuwenxing — CHANGES_REQUESTED** [review]

**Review by jiaoew1991 — APPROVED** [review]
/lgtm

**Pull request merged**